relay/kusama/src/lib.rs

```diff
@@ -2825,6 +3059,75 @@ mod remote_tests {
 		ext.execute_with(|| Runtime::on_runtime_upgrade(UpgradeCheckSelect::PreAndPost));
 	}
 
+	#[tokio::test]
+	async fn next_inflation() {
+		use hex_literal::hex;
+		sp_tracing::try_init_simple();
+		let transport: Transport =
+			var("WS").unwrap_or("wss://rpc.dotters.network/kusama".to_string()).into();
+		let mut ext = Builder::<Block>::default()
+			.mode(Mode::Online(OnlineConfig {
+				transport,
+				hashed_prefixes: vec![
+					// entire nis pallet
+					hex!("928fa8b8d92aa31f47ed74f188a43f70").to_vec(),
```

Comment:
Downloading the state lazily would be much smarter -  no need to declare upfront what keys are required.

Reply:
yeah, in this case it is fine as nis pallet is mostly empty.

remote-ext still doesn't have lazy download, right? or are you suggesting it because it now has it :o?

Reply:
>or are you suggesting it because it now has it :o?

Nah i think it still has not. But could be nice 🙃